app/views/courses/EnrollmentsView.coffee

```diff
@@ -59,7 +62,12 @@ module.exports = class EnrollmentsView extends RootView
     @listenToOnce @classrooms, 'sync', @onceClassroomsSync
     @supermodel.trackRequest @classrooms.fetchMine()
     @prepaids = new Prepaids()
-    @supermodel.trackRequest @prepaids.fetchByCreator(me.id)
+    @supermodel.trackRequest @prepaids.fetchMineAndShared(me.id)
+    @listenTo @prepaids, 'sync', ->
+      @prepaids.each (prepaid) =>
+        prepaid.creator = new User()
+        if prepaid.get('creator') isnt me.id
+          @supermodel.trackRequest prepaid.creator.fetchCreatorOfPrepaid(prepaid)
```

Comment:
This is one of those cases where I don't store the information because the template doesn't use it in this case. I'd love to hear your thoughts on how to go about this kind of thing. Better to store correct/less confusing/less breakable information even if it's not currently being used?

Line 119 of the template is where we avoid using that information.